README.md

```diff
@@ -0,0 +1,85 @@
+# ![nfcore/neutronstar](docs/images/nfcore-neutronstar_logo.png)
+
+**De novo assembly pipeline for 10X linked-reads.**
+
+[![Build Status](https://travis-ci.org/remiolsen/nf-core-neutronstar.svg?branch=master)](https://travis-ci.org/remiolsen/nf-core-neutronstar)
```

Comment:
I think this is sorted out, I'll wait for the build to complete. Does that mean in the future we are moving away from Travis? Because `.travis.yml` is still in the template code.

Reply:
Its still there, linting will get an update very soon with 1.8 to ask for removal but if you release earlier than V 1.8 of nf-core/tools, this won't be flagged 👍  

In the future we can remove it in the next release, but I guess this is fine now :-)